doc/example/getting_started.ipynb

```diff
@@ -639,9 +647,7 @@
     }
    },
    "source": [
-    "The plot shows that seven parameters are identifiable, since the likelihood is tightly centered around the optimal parameter. Two parameters (`k_exp_hetero` and `k_imp_homo`) cannot be constrained by the data.\n",
-    "\n",
-    "Furthermore pyPESTO allows to visualize confidence intervals directly via"
+    "Furthermore, pyPESTO allows to visualize approximate confidence intervals directly via `profile_cis`. Confidence intervals are computed by finding parameter values for which the log posterior ratio is above the approximate threshold assuming a $\\chi^2$ distribution of the likelihood test statistic. The plot shows that both model parameters are identifiable, since the confidence regions are finite."
```

Comment:
> The plot shows that both model parameters are identifiable, since the confidence regions are finite.

For parameters with finite bounds, the computed confidence region would always be finite, independent of identifiability, wouldn't it?

Reply:
> I'm unsure whether that's important to add here or even how I would state it. 

I'd find it important to clarify it, to not encourage users to do their "identifiability analysis" via `is_identifiable = ~np.any(np.isinf(compute_ci(...)))` which would claim identifiability for every constrained parameter.

Maybe something like that the profile crosses the threshold on both sides of the MLE, and therefore, the parameter is considered identifiable? (ignoring potential multiple optima)

> But the statement that it's identifiable because the confidence intervals are finite is definitely true, no?

I'd say it depends on how the confidence intervals are computed. If we just stop at the bounds, then no, not necessarily. If we extend the bounds as you pointed out, then yes.